app/views/shared/_email_header.html.erb

```diff
@@ -0,0 +1,9 @@
+<table class="header">
+  <tr>
+    <td>
+      <%= link_to root_url, class: 'header-logo' do %>
+        <%= image_tag 'logo-black.png', class: 'header-logo__image' %>
```

Comment:
I confirmed styling and image is working in gmail on safari. One strange thing is that the base64 for the image appears as the teaser of the email content. We'll want to fix that somehow
<img width="971" alt="screen shot 2016-03-06 at 12 29 47 am" src="https://cloud.githubusercontent.com/assets/1417199/13552686/03cbfac4-e333-11e5-85df-5928180bb608.png">